### DISCUSSION TITLE
posts/new-blog

### DISCUSSION THREAD

**Discussion by giscus** [EarthMessenger/blog.earthmessenger.xyz#6] [giscus]
# 2021/07/21/new-blog

新博客 建立啦！ 撒花！ 🎉

https://blog.earthmessenger.xyz/2021/07/21/new-blog.html#giscus

**Comment by zzyhzyy** [comment]
%%%

**Comment by fat-old-eight** [comment]
%%%

**Comment by xxEray** [comment]
加一下友链谢谢啦（我已经加好了 <https://xxeray.gitlab.io/links/>）

- link: <https://xxeray.gitlab.io/guestbook/links/>
- description: `OI 技术向博客。`
- avatar: <https://xxeray.gitlab.io/images/avatar/64x64.jpg>

**Comment by xyang333** [comment]
%%%

**Comment by zzyhzyy** [comment]
？



---Original---
From: ***@***.***&gt;
Date: Wed, Jul 9, 2025 13:18 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [EarthMessenger/blog.earthmessenger.xyz] posts/new-blog(Discussion #6)




 
%%%
